odyssey-lib/src/main/java/com/codebutler/odyssey/lib/retro/RetroDroid.kt

```diff
@@ -145,23 +141,11 @@ class RetroDroid(private val context: Context, coreFile: File) :
         return saveRam
     }
 
-    fun start() {
-        val avInfo = systemAVInfo
-        if (timerTask != null || avInfo == null) {
-            return
-        }
-        val timerTask = object : TimerTask() {
-            override fun run() {
-                retro.run()
-            }
-        }
-        timer.scheduleAtFixedRate(timerTask, 0, 1000L / avInfo.timing.fps.toLong())
-        this.timerTask = timerTask
-    }
-
-    fun stop() {
-        timerTask?.cancel()
-        timer.purge()
+    /**
+     * Runs a single loop of the game. This should be scheduled with Choreographer.
+     */
+    fun run() {
```

Comment:
The Choreographer is available as a singleton, but I didn't want to put that Android specific component here in the lib, so I changed this to let the host activity manage the actual loop.  

If you're comfortable with it, we can move the choreographer frame callback and looping work here and keep the start/stop semantics.